docs/docs/creating-a-transformer-plugin.md

```diff
@@ -27,7 +27,7 @@ Just like a source plugin, a transformer plugin is a normal NPM package. It has
 
 `gatsby-transformer-yaml` is transformer plugin that looks for new nodes with a media type of text/yaml (e.g. a .yaml file) and creates new YAML child node(s) by parsing the YAML source into JavaScript objects.
 
-As an example, let's partially rebuild a simplified `gatsby-transformer-yaml` directly in an example site. Let's say we have a default gatsby starter site which includes a `src/data/example.yml` file:
+Checkout this example of rebuilding a simplified `gatsby-transformer-yaml` directly in an example site. For this example, you have aa default gatsby starter site which includes a `src/data/example.yml` file:
```

Comment:
```suggestion
Check out this example of rebuilding a simplified `gatsby-transformer-yaml` directly in a site. Say you have a default Gatsby starter site which includes a `src/data/example.yml` file:
```

Reply:
This is a great change - thank you! I wasn't sure the best way to simplify it - your suggestion is fantastic.